### DISCUSSION TITLE
Improving Tables Extraction and Retrieval!

### DISCUSSION THREAD

**Discussion by abrahimzaman360** [safevideo/autollm#233] [Q&A]
Except for tables everything is working fine!
How to retrieve tables accurately.

**Comment by abrahimzaman360** [comment]
I want to get sources information like filename, document title for metadata extraction!
Please make it possible or give a clue if possible!
Thanks

**Comment by fcakyon** [comment]
Hey @AbrahimZaman360, autollm is using llama-index and litellm classes under the hood. You can check their documentation and improve the autollm functions. We are open to such PRs 👍

**Reply by abrahimzaman360** [reply]
If I edit the source autollm, then I've deploy code in docker.
